src/scenes/characters/character.gd

```diff
@@ -17,6 +17,7 @@ func interact() -> void:
     if (!GameState.is_in_interaction()):
         GameState.set_is_in_dialogue(true);
         var dialog = Dialogic.start(npc + str(dialog_index));
+        Dialogic.set_variable("CatFrogTimeline", 0);
```

Comment:
What is this for?

Reply:
it is useless sorry. it has been removed